### PR TITLE
Make the Kernel timer the default clock for Zephyr

### PR DESCRIPTION
**Pull request by erlingrj**
See https://github.com/lf-lang/reactor-c/pull/400

### CODE REVIEW

**Review by lhstrh — APPROVED** [review]
Looks good!

**Comment by erlingrj** [comment]
I will merge it once the corresponding PR is merged and update the reactor-c pointer

**Pull request merged**
